lms/templates/courseware/accordion.html

```diff
@@ -40,9 +40,12 @@
                 ## a status icon
                 <%
                     if section.get('due') is None:
-                        data_string = ''
+                        data_string = _(section['format'])
                     else:
-                        data_string = _('due {date}')
+                        if 'proctoring' in section:
+                            data_string = _('due {date}')
+                        else:
+                            data_string = _(section['format'] + " due {date}")
```

Comment:
I'm happy to make that change. Right, given that we don't translate this descriptor, my work doesn't make sense.